guardian/commercial-templates
# node-sass is deprecated and is being replaced with dart-sass

## What does this change?
[node-sass](https://www.npmjs.com/package/node-sass) is used as a dev dependency, but is now deprecated and not supported. This request swaps that dependency to sass which now uses dart-sass, which is compatible with node-sass.

https://www.npmjs.com/package/sass

sass.renderSync appears to be fully supported
https://github.com/sass/dart-sass/blob/master/README.md#javascript-api

We use sass.types here.
https://github.com/guardian/commercial-templates/blob/main/lib/css.js#L34

I couldn't see anything explicit within dart sass docs to suggest this was or wasn't supported. Testing locally showed no errors though and still functioned ok.

## How to test
Run npm i
Run npm run preview or npm run build

Previews should render all ad previews with no change and build script should build minified builds with no errors.
